SwinjectTests/LoadAwareAssembly.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  LoadAwareAssembly.swift
+//  Swinject
+//
+//  Created by mike.owens on 12/9/15.
+//  Copyright © 2015 Swinject Contributors. All rights reserved.
+//
+
+import Swinject
+
+
+class LoadAwareAssembly: AssemblyLoadAwareType {
+    var onLoad: (ResolverType) -> Void
+    var loaded = false
+    
+    init(onLoad: (ResolverType) -> Void) {
+        self.onLoad = onLoad
+    }
+    
+    func assemble(container: Container) {
+        container.register(AnimalType.self) { r in
+            return Cat(name: "Bojangles")
+        }
+    }
+    
+    func loaded(resolver: ResolverType) {
+        onLoad(resolver)
+        loaded = true
+    }
+}
+
+
```

Comment:
Trailing Newline Violation: Files should have a single trailing newline. (trailing_newline)